Tests/MongoSwiftBenchmarks/BsonBenchmarks.swift

```diff
@@ -3,16 +3,16 @@ import Foundation
 import XCTest
 
 let basePath = XCTestCase.getSpecsPath()
-let flatBsonFile = URL(fileURLWithPath: basePath + "/flat_bson.json")
+let flatBSONFile = URL(fileURLWithPath: basePath + "/flat_bson.json")
```

Comment:
Should these remain lowercase, since they're essentially variable/property names?

Or did we decide that "bson" would only be lowercase if it was the first segment of a variable/property name? I think we may have decided that when I brought up the `fromBSON` argument name remaining as-is.

Reply:
I think the correct thing to do here is to keep this capitalized. The idea was to keep with camelcase, stay consistent and do so as simply as possible:

1. When 'bson' needs to be lowercased, it is 'bson'.
2. When 'bson' needs to be uppercased, it is 'BSON'.

Those variables can be immediately recognized as variable/property names because their very first word (`flat`) is lowercased. If the `flat` was not there, it would be `bsonFile`. Since the `flat` is there, we have `flatBSONFile`.

Let me know what you think about this.

Reply:
I think what May has here is good. basically, all the letters in BSON should always be a single case. 

this is consistent with Apple's code - for example, note [this method](https://github.com/apple/swift-corelibs-foundation/blob/3995b2cde03b7d12e3e8f8f783b6e43e6322d60a/Foundation/JSONSerialization.swift#L311) is `serializeJSON`, not `serializeJson`.